comid/integrityregisters.go

```diff
@@ -0,0 +1,134 @@
+// Copyright 2024 Contributors to the Veraison project.
+// SPDX-License-Identifier: Apache-2.0
+package comid
+
+import (
+	"encoding/json"
+	"fmt"
+	"strconv"
+
+	"github.com/veraison/swid"
+)
+
+const TextType = "text"
+
+// IRegisterIndex is the interface to hold register index
+type IRegisterIndex interface{}
+
+// IntegrityRegisters holds the Integrity Registers
+type IntegrityRegisters struct {
+	Register map[IRegisterIndex]Digests
```

Comment:
`Register` doesn't need to be public.  (Hint: in VSC you can just "Rename Symbol" to some private name - e.g., `m`, `v`, `val`, as suggested previously.)

```suggestion
	m map[IRegisterIndex]Digests
```